src/libraries/System.Net.Quic/tests/FunctionalTests/QuicTestBase.cs

```diff
@@ -40,10 +42,31 @@ public abstract class QuicTestBase : IDisposable
         public const int PassingTestTimeoutMilliseconds = 4 * 60 * 1000;
         public static TimeSpan PassingTestTimeout => TimeSpan.FromMilliseconds(PassingTestTimeoutMilliseconds);
 
-        public QuicTestBase(ITestOutputHelper output)
+        private static bool s_queueDelaySet;
+        private static int s_initMessage;
+
+        static unsafe QuicTestBase()
+        {
+            if (IsSupported)
+            {
+                QUIC_SETTINGS settings = default(QUIC_SETTINGS);
+                settings.IsSet.MaxWorkerQueueDelayUs = 1;
+                settings.MaxWorkerQueueDelayUs = (uint)TimeSpan.FromSeconds(2.5).TotalMicroseconds;
+                s_queueDelaySet = StatusSucceeded(MsQuicApi.Api.ApiTable->SetParam(null, QUIC_PARAM_GLOBAL_SETTINGS, (uint)sizeof(QUIC_SETTINGS), (byte*)&settings));
+            }
+        }
+
+        public unsafe QuicTestBase(ITestOutputHelper output)
         {
             _output = output;
-            _output.WriteLine($"Using {MsQuicApi.MsQuicLibraryVersion}");
+            if (Interlocked.Exchange(ref s_initMessage, 1) == 0)
```

Comment:
should this be visible in first failing test? It seems like with this we would try to spit it out to first test runs. If that succeeds it would get discarded and we would not see it on next failure. 
And yes, you would see if for every failed test but that should be rare IMHO :)

Or do I miss something? 

Alternatively we may spit it out to Console in the new static constructor. While that is not stored in XML & test result database we get the console almost in all cases. (and I'm not sure we actually store the detail in Kusto any more)

Reply:
Originally, this spammed all the tests, every one had an output line in the testResults.xml. Now it'll just print itself in the first test.
I don't think there's a way to know that the test failed so that we can print it just in the failing tests. And I don't think that any of the outputs gets stored in Kusto, only the exceptions and their details (message + stack trace).
And printing to console is not advisable in tests as some runners eat that up, e.g.: https://stackoverflow.com/questions/55258178/not-able-to-print-output-to-console-window-while-running-xunit-tests

And now I'm seeing, that we don't have testResults.xml from CI... Just the console.log. I might print it to both, console and output and we'll see.